dotcom-rendering/src/web/components/SecureSignupIframe.importable.tsx

```diff
@@ -283,6 +288,40 @@ export const SecureSignupIframe = ({
 		resetIframeHeight();
 	};
 
+	const addFontToIframe = (requiredFontNames: string[]) => {
```

Comment:
i think this is looking pretty nice. how would you feel about pulling this up to https://github.com/guardian/libs? this is an interesting problem that wasn't obvious to how to solve at first. would be a good one to make more generally available/public?

Reply:
That sounds great - I'll look at doing a PR in /libs. We've got some time pressure on this feature so I'd probably  need to merge in this PR and then do another PR to use the import from /libs when the update is available in DCR.

Reply:
ok great - yeah whenever it works for you